### PR TITLE
Add 'orderBy' option

### LINKED ISSUE #80: Add option to order alphabetically
What about a configuration option to change the default sorting? 
Personally I have some use cases where I'd love to see my projects ordered alphabetically, because I have a high number of projects and for me it is hard to find the right project in the dashboard.

### PR DESCRIPTION
**Pull request by lscheibel**
Adds orderBy option and 4 possible values: 'lastActivity', 'created', 'name', 'nameWithNamespace'

resolves #80

### CODE REVIEW

File: CONFIGURATION.md [hunk 1]
```diff
@@ -75,6 +75,9 @@ showUsers: false
 # The page title, or null to hide
 title: null
 
+# How to order the displayed projects. Accepts: lastActivity, created, name, nameWithNamespace
```
Comment (timoschwarzer) [inline CONFIGURATION.md:78]:
```suggestion
# Project display order. Possible values: lastActivity, created, name, nameWithNamespace
```

File: src/components/app.vue [hunk 2]
```diff
@@ -80,14 +80,33 @@
...
+          created: (a, b) => new Date(b.created_at).getTime() - new Date(a.created_at).getTime(),
+          name: (a, b) => a.name.localeCompare(b.name),
+          nameWithNamespace: (a, b) => a.name_with_namespace.localeCompare(b.name_with_namespace),
+        }
+
+        let byMethod = sortMethods[Config.root.orderBy]
+
+        if (byMethod == null) {
+          console.warn(
+            `Config.orderBy: Could not resolve sort method, using 'lastActivity' instead.`,
```
Comment (timoschwarzer) [inline src/components/app.vue:95]:
```suggestion
            `Config.orderBy: Invalid configuration value "${Config.root.orderBy}", falling back to lastActivity`,
```

File: src/components/app.vue [hunk 3]
```diff
@@ -80,14 +80,33 @@
...
+            `Available methods are ${Object.keys(sortMethods).join(', ')}`
```
Comment (timoschwarzer) [inline src/components/app.vue:96]:
```suggestion
            `Possible values: ${Object.keys(sortMethods).join(', ')}`
```

**Review by timoschwarzer — CHANGES_REQUESTED** [review]

**Comment by timoschwarzer** [comment]
Would be nice if you'd implement an `orderByDesc: bool` option.

e.g.

```js
if (orderByDesc) {
  byMethod = (a, b) => sortMethods[Config.root.orderBy](b, a)
}
```

**Comment by lscheibel** [comment]
All done

File: CONFIGURATION.md [hunk 4]
```diff
@@ -75,6 +75,13 @@ showUsers: false
 # The page title, or null to hide
 title: null
 
+# Project display order. Possible values: lastActivity, created, name, nameWithNamespace
+# Note that this only changes the displayed order and not the order in which they will be fetched.
+orderBy: lastActivity
+
+# Wether to order displayed projects descending
```
Comment (timoschwarzer) [inline CONFIGURATION.md:82]: "Sort in descending order"

**Review by timoschwarzer — CHANGES_REQUESTED** [review]

**Comment by lscheibel** [comment]
Changed wording

**Pull request merged**